addons/mrp_account/models/mrp_production.py

```diff
@@ -18,9 +18,16 @@ def _compute_show_valuation(self):
         for order in self:
             order.show_valuation = any(m.state == 'done' for m in order.move_finished_ids)
 
+    def get_sol_analytic_distribution(self):
+        return {}
+
     @api.depends('bom_id', 'product_id')
     def _compute_analytic_distribution(self):
         for record in self:
+            sol_analytic_distribution = record.get_sol_analytic_distribution()
+            if sol_analytic_distribution:
+                record.analytic_distribution = sol_analytic_distribution
+                continue
             if record.bom_id.analytic_distribution:
```

Comment:
```suggestion
            elif record.bom_id.analytic_distribution:
```
I know it will impact the blame, but never mind, it's a bit weird otherwise